src/Pages/Login/Login.js

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import CommonInput from '../../Components/CommonInput/CommonInput';
+import './Login.scss';
+
+class Login extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      phoneValue: '',
+      passwordValue: '',
+      isLogin: false,
+      isWarning: [false, false],
+    };
+  }
+
+  checkInputLenght = () => {
+    const { phoneValue, passwordValue } = this.state;
+    this.setState({
+      isLogin: phoneValue.length > 0 && passwordValue.length > 0 ? true : false,
+    });
+  };
+
+  handleInput = e => {
+    const { name, value } = e.target;
+    this.setState(
+      {
+        [name]: value,
+      },
+      () => {
+        this.checkInputLenght();
+      }
+    );
+  };
+
+  handleLogin = () => {
+    fetch('http://10.58.2.121:8000/users/signin', {
+      method: 'POST',
+      body: JSON.stringify({
+        phone_number: this.state.phoneValue,
+        password: this.state.passwordValue,
+      }),
+    })
+      .then(res => res.json())
+      .then(res => {
+        if (res.result === 'SUCCESS!') {
+          this.goToMain();
+        } else if (res.result === 'INVALID USER') {
+          // console.log(this.state.isWarning);
+          this.setState({
+            isWarning: [true, false],
+          });
+        }
+      });
+  };
+
+  goToSign = () => {
+    this.props.history.push('/Sign');
+  };
+
+  goToMain = () => {
+    this.props.history.push('/');
+  };
+
+  render() {
+    const { isLogin, phoneValue, isWarning } = this.state;
+    console.log(isWarning);
+    return (
```

Comment:
```suggestion
  render() {
    const { phoneValue, passwordValue, isWarning } = this.state;

    let activeLogin = phoneValue.length > 0 && passwordValue.length > 0 ? true : false;

    return (
```
이런식으로 변수로 만들어서 사용할 수 있습니다!!